### ISSUE TITLE
Improve UX/UI for cv mailers

### ISSUE THREAD

**Issue by Kartikay304** [CircuitVerse/CircuitVerse#2658]
**Is your feature request related to a problem? Please describe.**

At the moment, the appearance of CV emails is minimalistic, and not very interactive. I think we should make some UX changes to CV mailers.

**Current Design**

![Screenshot from 2021-11-26 22-55-32](https://user-images.githubusercontent.com/40604582/143868109-be92c2a3-9f91-4f6d-98b7-ba72a1bd190a.png)

**Are you working on this? (Yes/No)**
Yes

**Comment by vedant-jain03** [comment]
@Kartikay304 It will be great if you put circuitverse logo on the design.

**Comment by Kartikay304** [comment]
> @Kartikay304 It will be great if you put circuitverse logo on the design.

Yeah, when I will make the design for cv mailers then, I will add the logo to it.

**Comment by ASGDMN** [comment]
I would like to work on this issue. 
We just have to add the logo and make it look good. I saw the code and all user_mailer are very minimalist so should I change all or just welcome email?

**Comment by Kartikay304** [comment]
> I would like to work on this issue. We just have to add the logo and make it look good. I saw the code and all user_mailer are very minimalist so should I change all or just welcome email?

Sure go ahead!, not just welcome mail change all of them.

**Comment by ASGDMN** [comment]
I think this one is fixed. This is what I got which is different from what you got. 
![image](https://user-images.githubusercontent.com/97045861/152412404-e2d182eb-b3b5-41b6-aa8c-ab0dc05256dd.png)
